Public/Src/Engine/Scheduler/Scheduler.cs

```diff
@@ -4857,10 +4907,17 @@ private void ProcessFileChanges(LoggingContext loggingContext, SchedulerState sc
             if (m_configuration.Schedule.InputChanges.IsValid)
             {
                 inputChangeList = InputChangeList.CreateFromFile(
-                    loggingContext, 
+                    loggingContext,
                     m_configuration.Schedule.InputChanges.ToString(Context.PathTable),
                     m_configuration.Layout.SourceDirectory.ToString(Context.PathTable),
                     DirectoryTranslator);
+
+                // Only master maintain the list
+                if (!IsDistributedWorker && "DynamicCodeCov".Equals(Environment.GetEnvironmentVariable("[Sdk.BuildXL]qCodeCoverageEnumType")))
+                {
+                    m_affectedOutputList = new AffectedOutputList(Context.PathTable, m_fileContentManager);
+                    m_configuration.Sandbox.AreAllInputsAffected = m_affectedOutputList.InitialAffectedOutputList(inputChangeList, Context.PathTable);

```

Comment:
>m_configuration.Sandbox.AreAllInputsAffected [](start = 20, length = 44)

Typically we use the global configuration object for things that are set on the command line for how the overall app/engine/scheduler are supposed to operate. This doesn't look to be user configurable and is only consumed within Scheduler.cs. It seems like it would be a better fit to just have it as local state within the scheduler.